src/StrategyEngine.cpp

```diff
@@ -0,0 +1,96 @@
+#include <unistd.h>
+#include <iostream>
+#include "StrategyEngine.h"
+
+StrategyEngine::StrategyEngine() {
+	std::cout << "Computing strategy" << std::endl;
+	return;
+}
+
+decisions StrategyEngine::getchoice(int dealercard, std::vector<int> playercards) {
+	
+	int numofcards = playercards.size();
+	int i = 0;
+	int playertotal = 0;
+	bool soft = false;
+	bool split = false;
+	if (numofcards == 2) {
+		if (playercards[0] == playercards[1]) {
+			split = true;
+		}
+	}
+	while (i < numofcards) {
+		if ((playercards[i] == 11) && (playertotal + playercards[i] <= 21)) {
+			soft = true;
+		}
+		playertotal = playercards[i] + playertotal;
+		i++;
+		if ((playertotal > 21) && (soft)) {
+			playertotal = playertotal - 10;
+			soft = false;
+		}
+		std::cout << playertotal << std::endl;
+		if (playertotal > 21) {
+			std::cout << "You Lose!" << std::endl;
```

Comment:
@Eagleeye101 If you lose, it will return STAND right? Maybe we can also have a LOSE enum option.
You can have in this case return LOSE; directly